src/fastcs_odin/util.py

```diff
@@ -17,11 +18,18 @@ class AdapterType(str, Enum):
     EIGER_FAN = "EigerFanAdapter"
 
 
+class metadataModel(BaseModel):
```

Comment:
I think this would be a better name
```suggestion
class OdinParameterMetadata(BaseModel):
```